docs/resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/change_mutable_qos_pub1.cpp

```diff
@@ -0,0 +1,96 @@
+#include <memory>
```

Comment:
Add license here. Same in all the other code files

Reply:
Done